src/swap/utils.ts

```diff
@@ -34,11 +35,31 @@ export function getQuote(params: {
   return getFixedOutputSwapQuote(params);
 }
 
+function validateQuotes(promises: Promise<SwapQuoteWithPool>[]) {
```

Comment:
```suggestion
/**
* Suppress .... error unless all of the pools has it ....
* @returns the resolved .... after filtering out .....
*/
function validateQuotes(promises: Promise<SwapQuoteWithPool>[]): SwapQuoteWithPool {
```
Let's add a comment and explain what it does, and what it returns for future us 😄 . Also let's add an explicit return type

Reply:
Added a description for this on #11